tests/S3/ObjectCopierTest.php

```diff
@@ -105,51 +105,64 @@ public function testDoesCorrectOperationAsynchronouslyWithPathStyle(
         $this->assertTrue($this->mockQueueEmpty());
     }
 
-    public function getCopyTestCases()
+    private function getSmallPutObjectMockResult()
     {
         $smallHeadObject = new Result(['ContentLength' => 1024 * 1024 * 6]);
         $putObject = new Result();
+
+        return [$smallHeadObject, $putObject];
+    }
+    private function getMultipartMockResults()
```

Comment:
Not a blocker, but there should be an empty line between this method and the previous one.